app/views/user/birth_records/index.html.erb

```diff
@@ -3,8 +3,11 @@
 </section>
 <% if @birth_records.any? %>
   <% @birth_records.each do |record| %>
-    <%= link_to 'remove', remove_user_birth_record_path(record), class: 'pill pill-secondary', method: :post %>
-    <%= render partial: 'shared/birth_records/card', object: record %>
+    <div class="user-birth-record" data-id="<%= record.to_param %>">
+      <%= link_to 'share', new_user_share_path(share: { birth_record_id: record.to_param }), class: 'pill pill-secondary' %>
```

Comment:
i think this needs to be on the birthrecords/show view also

Reply:
shouldnt this be a new_birth_record_share ? you're sharing the birth record not the user.

Reply:
It's a weirdness of the rails naming convention. Automatically generated from `user/share_controller#new`. There might be a better name for the model than `Share`, I'm open to suggestions. It should capture the meaning from 'User A consents to share record B with recipient C'. 

Maybe `ConsentToShare` rather than `Share`? It would read nicely as `user/consent_to_share#revoke` and `new_user_consent_to_share_path`.